internal/pkg/scaffold/olm-catalog/config.go

```diff
@@ -78,9 +79,17 @@ func (c *CSVConfig) setFields() error {
 
 	if len(c.CRDCRPaths) == 0 {
 		paths, err := getManifestPathsFromDir(scaffold.CRDsDir)
-		if err != nil {
+		if err != nil && !os.IsNotExist(err) {
 			return err
 		}
+		if os.IsNotExist(err) {
+			log.Infof(`Default CRDs dir "%s" does not exist. Omitting field spec.customresourcedefinitions.owned from CSV.`, scaffold.CRDsDir)
+			return nil
+		}
+		if len(paths) == 0 {
+			log.Infof(`Default CRDs dir "%s" is empty. Omitting field spec.customresourcedefinitions.owned from CSV.`, scaffold.CRDsDir)
+			return nil
+		}
```

Comment:
If we add more logic near the bottom of this function in the future, there will be an edge case where that logic is skipped.

Since these are valid-ish cases, should we fall through to the `return nil` in line 123 instead of returning immediately here?

```go
if os.IsNotExist(err) {
	log.Infof(...)
} else if len(paths) == 0 {
	log.Infof(...)
} else {
	c.CRDCRPaths = paths
}
```